hack/tag-release.sh

```diff
@@ -21,6 +21,21 @@ if [[ ! "${VERSION}" =~ ^([0-9]+[.][0-9]+)[.]([0-9]+)(-(alpha|beta)[.]([0-9]+))?
   exit 1
 fi
 
+BRANCH=$(git branch --show-current)
+
+if [[ ! "${BRANCH}" =~ ^(release-)([0-9]+[.][0-9]+)$ ]]; then
+    echo "Automatic tag creation must take place on a release branch."
+    exit 1
+fi
+
+BRANCH_MAJ_MIN=${BRANCH#release-}
+VERSION_MAJ_MIN=$(echo ${VERSION} | sed -Ee 's/-(alpha|beta)\.[0-9]+$//' | sed -Ee 's/\.[0-9]+$//')
```

Comment:
First remove something like `-alpha.0` if it exists, second remove the patch version.

Reply:
could be easier/clearer to just grep the "vX.Y" tbh ? but lgtm anyway